imports/startup/server/index.coffee

```diff
@@ -274,15 +274,29 @@ api.addRoute 'recentDescriptorMentions',
 @apiName recentAgents
 @apiGroup agent
 @apiParam {String} promedFeedId=all The ProMED-mail regional feed to query
+@apiParam {ISODateString} start=2 weeks ago
+@apiParam {ISODateString} end=today
 ###
 api.addRoute 'recentAgents',
   get: ->
     promedFeedId = @queryParams.promedFeedId
     if promedFeedId == 'null'
       promedFeedId = null
-    page = @queryParams.page
-    pp = @queryParams.pp
-    offset = page * pp
+    start = moment(@queryParams.start or moment().subtract(2, 'weeks'))
+    end = moment(@queryParams.end or moment())
+    # Round dates up to the day end so that they are cached
+    start.set(
+      hour: 23
+      minute: 59
+      second: 59
+      millisecond: 0
+    )
+    end.set(
+      hour: 23
+      minute: 59
+      second: 59
+      millisecond: 0
+    )
```

Comment:
Maybe we should remove the parenthesis so we are inline with the style guide? Also in `recentAgents.coffee` on line 122